torch/distributed/distributed_c10d.py

```diff
@@ -1759,15 +1761,25 @@ def broadcast_object_list(object_list, src=0, group=None):
     else:
         object_sizes_tensor = torch.empty(len(object_list), dtype=torch.long)
 
+    # Current device selection.
+    # To preserve  backwards compatibility, dist_device is default to None in
```

Comment:
duplicated space between `preserve` and  `backwards`

Also, please update `dist_device` accordingly if the arg name changed.

Reply:
done